[WIP] PSL Year in Review (#23)
PSLmodels/blog: _posts/2020-12-23-2020_year_in_review.md

---
 toc: true
 layout: post
 description: Highlights from PSL in 2020
 categories: [PSL, PSL-Foundation]
 author: Jason DeBacker
 title: "2020: A Year in Review"
---

This year has been one to forget!

> But 2020 did have it's bright spots, especially in the PSL community.

Reviewer: ```suggestion
But 2020 did have its bright spots, especially in the PSL community.
```

This post reviews some of the highlights from the year.

The Library was able to welcome two new models to the catalog this year: [microdf](https://github.com/PSLmodels/microdf) and [OpenFisca-UK](https://github.com/PSLmodels/openfisca-uk).

> mircodf provides a number of useful tools for use with economic survey data.

Reviewer: ```suggestion
microdf provides a number of useful tools for use with economic survey data.
```
also should we code-font the packages?

> OpenFisca-UK builds off the [OpenFisca](https://openfisca.org/en/) platform, offering a microsimulation models for tax and benefits programs in the UK.

Reviewer: ```suggestion
OpenFisca-UK builds off the [OpenFisca](https://openfisca.org/en/) platform, offering a microsimulation models for tax and benefit programs in the UK.
```

> In addition, four new models were added to the Library as incubating projects.  The [ui-calculator](https://github.com/PSLmodels/ui_calculator) model has recieved a lot of attention this year in the U.S., as it provides the capability to calculate unemployment insurance benefits across U.S. states, a major mode of delivering financial relief to individual during the COVID crisis. 

Reviewer: ```suggestion
In addition, four new models were added to the Library as incubating projects.  The [ui-calculator](https://github.com/PSLmodels/ui_calculator) model has received a lot of attention this year in the U.S., as it provides the capability to calculate unemployment insurance benefits across U.S. states, a major mode of delivering financial relief to individuals during the COVID crisis. 
```

> [PCI-Outbreak](https://github.com/PSLmodels/PCI-Outbreak) directly relates to the COVID crisis, using maching learning and natural language processing to estimate the true degrees of COVID cases in China.

Reviewer: ```suggestion
[PCI-Outbreak](https://github.com/PSLmodels/PCI-Outbreak) directly relates to the COVID crisis, using machine learning and natural language processing to estimate the true degrees of COVID cases in China.
```
